tests/unit/cython/types_testhelper.pyx

```diff
@@ -52,11 +52,11 @@ def test_datetype(assert_equal):
     # deserialize
     # epoc
     expected = 0
-    assert_equal(deserialize(expected), datetime.datetime.utcfromtimestamp(expected))
+    assert_equal(deserialize(expected), datetime.datetime.fromtimestamp(expected, tz=datetime.timezone.utc))
 
     # beyond 32b
     expected = 2 ** 33
-    assert_equal(deserialize(expected), datetime.datetime(2242, 3, 16, 12, 56, 32))
+    assert_equal(deserialize(expected), datetime.datetime(2242, 3, 16, 12, 56, 32, tz=datetime.timezone.utc))
```

Comment:
Note that we're dealing with a datetime constructor here rather than a utcfromtimestamp() invocation.  We don't _need_ to replace this in order to handle any deprecation concerns.

More to the point: the datetime coming out of deserialize() is naive so this actually introduces a test failure.  I believe this should stay as `assert_equal(deserialize(expected), datetime.datetime(2242, 3, 16, 12, 56, 32))`

Reply:
yeah, silly me! reverted this change.